toolkit/tools/internal/packagerepo/repoutils/repoutils.go

```diff
@@ -87,6 +88,8 @@ func RestoreClonedRepoContents(cloner repocloner.RepoCloner, srcFile string) (er
 		}
 	}
 
+	timestamp.Stamp.RecordToCSV("Clone RPM repo", "Verify cloned repo contents")
```

Comment:
For internal libraries where we aren't directly calling `timestamp.InitCSV()`, I think it makes sense to check that `timestamp.Stamp` is initialized.

Reply:
Talking with Pawel earlier, we agreed that the `timestamp` tool should something that can fail without breaking the build. Agreed we should check this, but it may be enough to add logic inside the `timestamp` package so that we auto populate the global with an empty object. If we try to write to that object it just prints a warning and otherwise no-ops.